lib/src/main/java/com/basistheory/android/model/InputType.kt

```diff
@@ -0,0 +1,10 @@
+package com.basistheory.android.model
+
+import android.text.InputType as AndroidInputType
+
+enum class InputType(val androidInputType: Int, val isConcealed: Boolean = false) {
```

Comment:
there's a bunch of others we are not supporting, i guess we will only if ppl ask for them?

Reply:
Yeah, we were only going to support `textPassword` so even `numberPassword` is a bonus